assets/stylesheets/components/_app-content.scss

```diff
@@ -0,0 +1,3 @@
+.app-content {
```

Comment:
Ditto here: these styles should go in the layout as they affect how the page rendered. So `.app-content` can have a specific style for each layout.